pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -106,13 +111,42 @@ let
   wxGTK = wxGTK32;
   python = python3;
   wxPython = python.pkgs.wxPython_4_2;
+  thirdPartyPackagePath = "package.zip";
+  thirdPartyPackagesDrvs = map (pkg: pkg.override { inherit thirdPartyPackagePath python3; }) thirdPartyPackages;
+
+  thirdPartyPackagesJoined =
+    let
+      thirdParty = "$out/share/kicad/scripting";
```

Comment:
We could just duplicate this to reduce the amount of variables

Reply:
Marking resolved as this is done!